### ISSUE TITLE
How to input htmls in different folders?

### ISSUE THREAD

**Issue by Basign** [acss-io/atomizer#349]
My folder structure is like this:

root
├── cn
│   └── foo
│       └── bar.html
├── en
│   └── foo
│       └── bar.html
├── hk
│   └── foo
│       └── bar.html

I tried "/root/**/*.html", but it didn't work.

What is the correct pattern?

**Comment by tom76kimo** [comment]
If you are under the `root` folder, try `atomizer -o atomic.css ./**/*.html`, it works for me.

**Comment by Basign** [comment]
> If you are under the `root` folder, try `atomizer -o atomic.css ./**/*.html`, it works for me.

Thanks for your help!

What I want is to look up in the path recursively, which I found in `atomizer -h` is `-R`. Problem solved.

But I still got 2 questions. First, if I wrote `-R ./` to find in all folders, how to find `*.html` specifically but not files of other types?

Second, how to exclude `.git/` folder? According to guidance, I wrote `atomizer -o xx.css --exclude=[.git/] -R ./` or `atomizer -o xx.css --exclude=[.git] -R ./`, but it didn't work.

**Comment by tom76kimo** [comment]
Yeah.. atomizer seems not handle cases like these two you mentioned. For now, better use it with webpack or grunt to handle the issue.
